code/game/turfs/open/lava.dm

```diff
@@ -315,6 +315,9 @@
 		burn_living.adjust_fire_stacks(lava_firestacks * seconds_per_tick)
 		burn_living.ignite_mob()
 
+/turf/open/lava/can_cross_safely(atom/movable/crossing)
+	return HAS_TRAIT(crossing, TRAIT_LAVA_IMMUNE) || HAS_TRAIT(crossing, TRAIT_MOVE_FLYING)
```

Comment:
```suggestion
	return HAS_TRAIT(src, TRAIT_LAVA_STOPPED) || HAS_TRAIT(crossing, immunity_trait ) || HAS_TRAIT(crossing, TRAIT_MOVE_FLYING)
```

The trait to resist lava is stored in `immunity_trait `, liquid plasma uses a different one.